templatestore/frontend/src/pages/templateScreen.js

```diff
@@ -311,16 +321,13 @@ class TemplateScreen extends Component {
                     </div>
                     <div>
                         <label className={styles.teLabel}>
-                            Choose a type :
+                            Render Mode :
                         </label>
-                        <select
+                        <input
                             readOnly
-                            className={styles.teButtons}
+                            type="text"
                             value={this.state.subTemplatesData[t].renderMode}
-                        >
-                            <option value="text"> Text </option>
-                            <option value="html"> HTML </option>
-                        </select>
+                        />
```

Comment:
this will be select only but options will be disabled
https://www.w3schools.com/tags/att_option_disabled.asp